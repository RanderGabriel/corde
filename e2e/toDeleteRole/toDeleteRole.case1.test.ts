import Utils from "../testUtils";
import { runCLI } from "../cliRunner";

it("should delete a role", async () => {
  const command = Utils.buildCommandWithConfigPath("toDeleteRole", "bot_case1.test.ts");
  const results = await cli.exec(command);
  expect(results.statusCode).toEqual(0);
  expect(results.stdout).toMatchSnapshot();
});
