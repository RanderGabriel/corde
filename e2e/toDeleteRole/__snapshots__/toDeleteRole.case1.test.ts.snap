// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should delete a role 1`] = `
"
 PASS   e2e\\\\toDeleteRole\\\\__cordeTest__\\\\bot_case1.test.ts   100ms
   âœ”  should delete a role   100ms

Test Files: 1 passed. 1 total
Tests:      1 passed. 1 total
Time:       100ms
"
`;
